src/lib.rs

```diff
@@ -58,6 +76,9 @@ impl Contract {
             authors: UnorderedMap::new(StorageKey::AuthorToAuthorPosts),
             communities: UnorderedMap::new(StorageKey::Communities),
             featured_communities: Vec::new(),
+            last_project_id: 0,
+            projects: UnorderedMap::new(StorageKey::Projects),
+            project_views: UnorderedMap::new(StorageKey::ProjectViews),
```

Comment:
What is the reason for using UnorderedMap instead of Vector to index projects/project_views?

Reply:
also for deletion.